core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -225,4 +230,26 @@ class PolicyManager {
         }
         return freeBuildings
     }
+
+    private fun triggerGlobalAlerts(policy: Policy) {
+        // Preferably this should go in the JSON file, but I don't feel at ease editing such basic data types as the Policy class yet.
+        var extraNotificationText = when (policy.name) {
+            // Yes, this extra space is intentional, see the policies.json file.
+            "Patronage  Complete" -> "This causes our influence with City-States to drop faster!"
+            else -> ""
+        }.tr()
+        if (extraNotificationText != "") {
+            extraNotificationText = "\n${extraNotificationText}"
+        }
+        for (civ in civInfo.gameInfo.civilizations) {
+            if (civ == civInfo) continue
+            val defaultNotificationText = 
+                if (civ.getKnownCivs().contains(civInfo)) {
+                    "[${civInfo.civName}] has adopted the [${policy.name}] policy"
+                } else {
+                    "An unknown civilization has adopted the [${policy.name}] policy"
+                }.tr()
```

Comment:
No, don't .tr() this because then it will be 'stuck' in the language of the notification generator instead of being translated for each player individually